src/cdk/tree/tree.ts

```diff
@@ -450,7 +450,7 @@ export class CdkTree<T, K = T> implements AfterContentChecked, CollectionViewer,
     } else if (this._expansionModel) {
       const expansionModel = this._expansionModel;
       this._getAllDescendants()
-        .pipe(take(1), takeUntil(this._onDestroy))
+        .pipe(takeUntil(this._onDestroy))
```

Comment:
This seems like a strange change - wouldn't this only want to take one look at the descendants? Would this emit multiple times over the course of the component?

Reply:
`_getAllDecendants` emits multiple times, but does only call `getDescendants` once per node

Reply:
Could you add a comment here (above `this._getAllDescendants()`) that explains what's happening here? 

My mental model here is that `_getAllDescendants` is an observable because descendants may change over time, but the `expandAll` and `collapseAll` operations happen at a single point in time. My expectation would be that you would still want `take(1)` here with `_getAllDescendants`  not emitting until it has accumulated all nodes.

Reply:
I've added a text blurb above the `_getAllDescendants` function instead, does that clarify things?